blocks/products/products.js

```diff
@@ -375,10 +375,11 @@ export default function decorate(block) {
 
       // listen to ProductCard change and update the first button accordingly
       mv.subscribe((card) => {
-        const link = col.querySelector('.button-container a');
-        if (link) {
-          link.href = card.url;
-        }
+        col.querySelectorAll('.button-container a').forEach((link) => {
+          if (link && link.href === 'http://') {
```

Comment:
do we want to update the link only when is set to `http://`, I mean we don't want to do this when the link starts with `http://`?
I've checked the link for the PR and the line below is not reached on the `/solutions/` page

Reply:
> do we want to update the link only when is set to http://
Yes, Word forces the user to provide a value for the url, it cannot be blank. This is why I choose "http://"

Another alternative maybe is "https://www.bitdefender.com/site/Store/buy" which is the base url of the store.

> I've checked the link for the PR and the line below is not reached on the /solutions/ page
Thanks, will fix this.

Reply:
Fixed, if the link is equal to "https://www.bitdefender.com/site/Store/buy" it will be recalculated dynamiclaly to point to the right product in the store. Otherwise it will keep it as it is.

It has been fixed on:
- https://fix-button-url--bitdefender--hlxsites.hlx.page/solutions/ where the actual bug is fixed
- https://fix-button-url--bitdefender--hlxsites.hlx.page/solutions/total-security where the button is dynamically reclculated.

Reply:
checked and it works fine